core/services/payment_methods/forms/PayPalSettingsForm.php

```diff
@@ -108,7 +115,10 @@ protected function checkForCredentialsErrors()
 
         if (empty($response_args['ACK'])) {
             EE_Error::add_error(
-                esc_html__('Your PayPal credentials could not be verified. Part of their response was missing.', 'event_espresso'),
+                esc_html__(
+                    'Your PayPal credentials could not be verified. Part of their response was missing.',
+                    'event_espresso'
+                ),
```

Comment:
+1 for this. Should we do the same on lines `105`, `139` and `168` ?

Reply:
yeah may as well tidy that up, thanks